app/services/idv/in_person/completion_survey_sender.rb

```diff
@@ -0,0 +1,21 @@
+module Idv
+  module InPerson
+    class CompletionSurveySender
+      ##
+      # @param [User] user
+      # @param [String] issuer
+      def self.send_completion_survey(user, issuer)
```

Comment:
just double checking - this first survey is for the remote enrollment?

Reply:
No, this is solely for in-person enrollment.